examples/example-gateway/config/production.yaml

```diff
@@ -90,3 +90,4 @@ grpc.clientServiceNameMapping:
   echo: echo
 router.whitelistedPaths:
   - /path/whitelisted
+service.shadow.env.override.enable: true
```

Comment:
why is this needed? if you have shadowRequestHeader in build.yaml, you override it otherwise you don't.

Reply:
As discussed with Tejas, Zanzibar is used by multiple teams(edge-gateway and presentation), etc. 
This shadow header can pass to downstream services as well but we don't want to enforce env override for them as well. So I enable it through config.